feature-libs/organization/account-summary/occ/public_api.ts

```diff
@@ -0,0 +1 @@
+export * from './account-summary-occ.module';
```

Comment:
adapters and model also can be exported, right?

Reply:
yes